docs/agents/sematext-agent/kubernetes/configuration.md

```diff
@@ -0,0 +1,46 @@
+| KUBERNETES_EVENTS_NAMESPACE | Designates a namespace for Kubernetes event watcher. By default all namespaces are watched for Kubernetes events and forwarded to event/log receivers. |
+| KUBERNETES_NAMESPACES | Defines the comma separated list of namespaces that are queried for Kubernetes resources such as pods or deployments. By default all namespaces are fetched. You can adjust specific namespaces such as `KUBERNETES_NAMESPACES=default,kube-system`. |
+| KUBERNETES_INTERVAL | Defines the collection interval for Kubernetes resources (default 10s) |
+| KUBERNETES_CLUSTER_ID | Uniquely identifies the cluster where agent is deployed |
+| KUBERNETES_KUBELET_AUTH_TOKEN |  Specifies the path for account service token |
+| KUBERNETES_KUBELET_CA_PATH | Determines the file path for the certificate authority utilized during TLS verification |
+| KUBERNETES_KUBELET_CERT_PATH | Determines the file path for the certificate file utilized during TLS verification |
+| KUBERNETES_KUBELET_KEY_PATH | Determines the file path for the private key utilized during TLS verification |
+| KUBERNETES_KUBELET_INSECURE_SKIP_TLS_VERIFY |  Indicates whether to skip TLS verification |
+| KUBERNETES_KUBELET_METRICS_PORT |  Specifies the port where kubelet Prometheus metrics are exposed (default 10250) |
+| **Process Monitoring** | |
+| PROCESS_ENABLED | Specifies if process metrics collection is enabled. To disable process metrics collector set `PROCESS_ENABLED=false`. |
+| **Network Monitoring** | |
+| NETRACER_ENABLED | Controls whether network topology collector is turned on. Network tracer is disabled by default. To enable network tracer set `NETRACER_ENABLED=true`. In order to capture network traffic from the host, you should start the container with host networking by passing the `--network=host` argument to Docker engine. This is only required when pcap network tracer is used. |
+| NETRACER_INPUT_FILTER, NETRACER_OUTPUT_FILTER | When specified, applies filtering expressions to all inbound/outbound packets (`NETRACER_INPUT_FILTER="dst port 8923 and tcp"`). |
+| NETRACER_INTERFACES | A comma-separated list of network interfaces to monitor. By default all interfaces are tracked. |
+| NETRACER_LOOPBACK | Determines whether loopback interfaces should be captured for network traffic. By default network packets are not captured from loopback interfaces. |
+| NETRACER_EBPF_TRACK_TCP | Determines whether TCP connections are tracked by ebpf tracer. By default TCP traffic statistics are collected. |
+| NETRACER_EBPF_TRACK_UDP | Determines whether UDP connections are tracked by ebpf tracer. By default UDP traffic statistics are collected. |  
+| **Troubleshooting Options** | |
+| CPUPROFILE | Name of the file where `pprof` CPU profile is dumped. When provided this turns on the CPU profiling and writes data to a given file. |
```

Comment:
Not really a docs thing, so feel free to ignore, but we should have probably called his CPU_PROFILE to follow the naming pattern used in other env vars (e.g. we use LOGGING_LEVEL, not LOGGINGLEVEL)

Reply:
The naming nomenclature is imposed by how Viper (the package we use for config management)  internally handles flags. For example, with `logging.level`, Viper is able to infer this is going to be a nested key in yaml/json config file:

```yaml
logging:
   level: debug
```

...but for env variables it replaces the `.` with `_`.